lisa/tools/python.py

```diff
@@ -49,17 +49,40 @@ def _install(self) -> bool:
         self.node.os.install_packages(package_name)
         return self._check_exists()
 
-    def install_packages(self, packages_name: str) -> None:
-        cmd_result = self.run(
-            f"install -q {packages_name}",
-        )
+    def install_packages(
+        self,
+        packages_name: str,
+        install_path=""
+    ) -> None:
+        node = self.node
+        cmd_line = f"install -q {packages_name}"
+
+        if install_path != "":
+            tagert_path = install_path + "/python_packages"
+            node.tools[Mkdir].create_directory(tagert_path, sudo=True)
+            tmp_path = install_path + "/tmp"
```

Comment:
call the variable name "cache_path" is more accurate. Avoid to use "temp", if the usage is clear.